tests/infrastructure/tls-configuration.go

```diff
@@ -49,38 +49,35 @@ import (
 
 var _ = DescribeInfra("tls configuration", func() {
 
-	var (
-		virtClient kubecli.KubevirtClient
-		cipher     *tls.CipherSuite
-	)
+	var virtClient kubecli.KubevirtClient
```

Comment:
nit: There is no need for this, just call it multiple times, it is a singleton.

Reply:
I would prefer to tackle this pattern globally.